hack/test-end-to-end.sh

```diff
@@ -256,11 +256,23 @@ wait_for_url "${API_SCHEME}://${API_HOST}:${API_PORT}/api/v1beta1/minions/${KUBE
 # add e2e-user as a viewer for the default namespace so we can see infrastructure pieces appear
 openshift admin policy add-role-to-user view e2e-user --namespace=default
 
+# grant service accounts roles in the default namespace
+# TODO: figure out how to add these automatically to namespaces not created from project templates
```

Comment:
This is gorpy, especially because the default namespace will not get created from a template, and we put stuff in there (registry, router, etc), so none of our service accounts will have permissions in that namespace without intervention like this. Any ideas?